src/Application.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Manalize project.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Manalize;
+
+use Symfony\Component\Console\Application as BaseApplication;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class Application extends BaseApplication
+{
+    const VERSION = '0.1.1';
```

Comment:
The symfony installer seems to use the next patch version (i.e `0.1/2-DEV`) however instead.
But for now I guess we're far from a simple patch version since last release :)